go/vt/mysqlctl/backup.go

```diff
@@ -125,6 +127,19 @@ func Backup(ctx context.Context, params BackupParams) error {
 		return vterrors.Wrap(err, "unable to get backup storage")
 	}
 	defer bs.Close()
+
```

Comment:
why not define scope here 
scope := Stats.Scope(
			stats.Component(stats.BackupStorage),
			stats.Implementation(backupstorage.BackupStorageImplementation))
and use it #133 and #154

Reply:
Hey @rsajwani the `stats` objects used on L133 and L154 have different scopes, so we cannot re-use them.

Reply:
ah missed that .. go it ..